test/unittests/skills/test_fallback_skill.py

```diff
@@ -0,0 +1,49 @@
+from unittest import TestCase, mock
+
+from mycroft.skills import FallbackSkill
+
+
+def setup_fallback(fb_class):
+    fb_skill = fb_class()
+    fb_skill.bind(mock.Mock(name='bus'))
+    fb_skill.initialize()
+    return fb_skill
+
+
+class TestFallbackSkill(TestCase):
+    def test_life_cycle(self):
+        """Test startup and shutdown of a fallback skill.
+
+        Ensure that an added handler is removed as part of default shutdown.
+        """
+        self.assertEqual(len(FallbackSkill.fallback_handlers), 0)
+        fb_skill = setup_fallback(SimpleFallback)
+        self.assertEqual(len(FallbackSkill.fallback_handlers), 1)
+        self.assertEqual(FallbackSkill.wrapper_map[0][0],
+                         fb_skill.fallback_handler)
+
+        fb_skill.default_shutdown()
+        self.assertEqual(len(FallbackSkill.fallback_handlers), 0)
+        self.assertEqual(len(FallbackSkill.wrapper_map), 0)
+
+    def test_manual_removal(self):
+        """Test that the call to remove_fallback() removes the handler"""
+        self.assertEqual(len(FallbackSkill.fallback_handlers), 0)
+
+        # Create skill adding a single handler
+        fb_skill = setup_fallback(SimpleFallback)
+        self.assertEqual(len(FallbackSkill.fallback_handlers), 1)
+
+        fb_skill.remove_fallback(fb_skill.fallback_handler)
+        # Both internal trackers of handlers should be cleared now
+        self.assertEqual(len(FallbackSkill.fallback_handlers), 0)
+        self.assertEqual(len(FallbackSkill.wrapper_map), 0)
```

Comment:
Nit pick as it's already tested in the lifecycle test, however I'm still wondering if we should check the length of the wrapper_map after adding the Fallback Skill in this test as well. 

I can't really see any way that it would happen, but if the length of the wrapper map never changed from 0 this would still pass.